common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/Collection.scala

```diff
@@ -0,0 +1,10 @@
+package uk.ac.wellcome.models.work.internal
+
+case class Collection(
+  label: Option[String],
+  path: String,
+) {
+  val paths: List[String] =
+    path.split("/").inits.toList.reverse.drop(0).map(_.mkString("/"))
```

Comment:
I have a feeling this doesn't do what you intended it to do - `drop(0)` is a noop and it looks like you'd want `drop(1)` instead. 

It doesn't look like you use `paths` anywhere else and really it's there to calculate `depth` - how about just doing
```
val depth: Int = 1 + path.count(_ == "/")
```

Reply:
Correct, I've removed it, when we need it I'll add tests